src/html-imports.js

```diff
@@ -694,7 +685,28 @@
     document.addEventListener('load', onLoadingDone, true /* useCapture */ );
     document.addEventListener('error', onLoadingDone, true /* useCapture */ );
   } else {
-    new Importer();
+    // Override baseURI so that imported elements' baseURI can be used seemlessly
+    // on native or polyfilled html-imports.
+    // NOTE: a <link rel=import> will have `link.baseURI === link.href`, as the link
+    // itself is used as the `import` document.
+    const native_baseURI = Object.getOwnPropertyDescriptor(Node.prototype, 'baseURI');
```

Comment:
By doing prototypical patching, you're ensuring this won't work on older Chrome/Safari that put accessors on instances. I'm not sure it's worth it. Instead, just putting `baseURI` on import script instances seems like it solves the issue more simply. I suggest trying each approach on Safari 8 to check.